app/Http/Controllers/HomeController.php

```diff
@@ -69,14 +69,14 @@ public function dateRange(Request $request): JsonResponse
     {
         try {
             $stringStart = e((string)$request->get('start'));
-            $start       = Carbon::createFromFormat('Y-m-d', $stringStart);
+            $start       = Carbon::createFromFormat('Y-m-d H:i:s', "$stringStart 00:00:00");
```

Comment:
You can also force the start of the day by using `Carbon::createFromFormat('!Y-m-d', $stringStart);`